src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String text, String fileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new java.io.FileWriter(fileName))) {
+            bufferedWriter.write(text);
+        } catch (IOException e) {
+            throw new RuntimeException("Error writing to file", e);

```

Comment:
Excellent use of RuntimeException to wrap the IOException. This adheres to the checklist's guideline to avoid ignoring exceptions and provides an informative message along with the exception object.